README.md

```diff
@@ -177,6 +177,20 @@ prevent shipping late on a Friday, but still want to allow marking merge request
 More than one embargo period can be specified. Any merge request assigned to her
 during an embargo period, will be merged in only once all embargoes are over.
 
+## Restricting the list of projects marge-bot considers
+
+By default marge-bot will work on all projects that she is a member of.
+Sometimes it is useful to restrict a specific instance of marge-bot to a subset
+of projects. You can specify a regexp that projects must match (anchored at the
+start of the string) with `--project-regexp`.
+
```

Comment:
Yup, your understanding is correct and it's the `self.project_regexp.match` that does it (the alternative would be `self.project_regexp.search`, [which does not anchor at the beginning](https://docs.python.org/2/library/re.html#re.RegexObject)).